commands/util.go

```diff
@@ -231,7 +238,7 @@ func getInstanceByName(ctx context.Context, dockerCli command.Cli, instance, con
 	if err != nil {
 		return nil, err
 	}
-	return driversForNodeGroup(ctx, dockerCli, ng, contextPathHash)
+	return driversForNodeGroup(ctx, dockerCli, ng, contextPathHash, ng.Nodes[0].Endpoint)
```

Comment:
This probably needs extra care, does `driversForNodeGroup` really need to take a concrete endpoint?